docs/pages/docs/getting-started/app-router.mdx

```diff
@@ -148,24 +148,45 @@ export const config = {
 
 ### `app/[locale]/layout.tsx` [#layout]
 
-The `locale` that was matched by the middleware is available via the `locale` param and can be used to configure the document language.
+The `locale` that was matched by the middleware is available via the `locale` param and can be used to configure the document language. Additionally, we can use this place to pass configuration from `i18n.ts` to Client Components via `NextIntlClientProvider`.
```

Comment:
Always somewhat of a balance to include necessary information in the getting started guide, but be rather to the point here. We can save many details for later IMO.

Reply:
In which part of the docs is there more information on this?

Reply:
The [global configuration docs](https://next-intl-docs-git-docs-client-components-first-next-intl.vercel.app/docs/usage/configuration) have all the details about this.

I'd prefer to be quite selective about which links to include in the getting started docs and therefore not link to this page at this point. Links tend to create branches of user flows where a developer is just trying to get something up and running.

In the latest iteration I've now removed all links that would create "branches" and instead only used them if prerequisite knowledge is required (e.g. what a dynamic segment like `[locale]` is in Next.js).

Once the user has finished the setup, we can help the developer continue with the links in the "Next steps" section.

Does that sound reasonable to you?

Reply:
I completely agree. I think that the getting started steps should be as simple as possible to allow users to get things working and then provide links later to allow for more exploration. Otherwise, as you said, users can get a bit distracted reading into things and perhaps confused.